src/core/lib/channel/channel_stack.h

```diff
@@ -109,6 +112,19 @@ struct grpc_channel_filter {
      See grpc_call_next_op on how to call the next element in the stack */
   void (*start_transport_stream_op_batch)(grpc_call_element* elem,
                                           grpc_transport_stream_op_batch* op);
+  /* Create a promise to execute one call.
+     If this is non-null, it may be used in preference to
+     start_transport_stream_op_batch.
+     If this is used in preference to start_transport_stream_op_batch, the
+     following can be omitted also:
+       - calling init_call_elem, destroy_call_elem, set_pollset_or_pollset_set
```

Comment:
Does this actually eliminate the need for the pollset calls?  I would have thought that that would be orthogonal, since we still need the pollset linkage until we eliminate iomgr in favor of EventEngine.  Or does the promise infrastructure have some other way of doing this pollset linkage?

Reply:
When this comes up, we'll either need to finish EventEngine or find some other way for the promise stuff to handle that - but either way I want this stake in the ground - individual promises will never need to deal with pollsets.